core/src/test/java/org/hibernate/ogm/backendtck/package-info.java

```diff
@@ -7,7 +7,7 @@
 /**
  * This package contains end-to-end tests which should be executed for every datastore module. If required, single
  * tests can be skipped on specific dialects by annotating them with
- * {@link org.hibernate.ogm.utils.SkipByGridDialect}.
+ * {@link org.hibernate.ogm.utils.SkipByGridModule}.
```

Comment:
This change seems obsolete now. It should mention SkipByGridDialect/SkipByDatastoreProvider instead.